examples/rest2/claim_positions.js

```diff
@@ -1,18 +1,17 @@
 'use strict'
 
 const Promise = require('bluebird')
+const { RESTv2 } = require('bfx-api-node-rest')
```

Comment:
would it be clearer to have it pointing to the wrapped version we offer in this repo?

```js
const { RESTv2 } = require('../../index')
```

Reply:
I don't think this is a good solution, since the `bitfinex-api-node` package only re-exports this class. As a result of refactoring we can see that this example does not depend on `bitfinex-api-node` package. 
Maybe this and similar examples should be completely moved to `bfx-api-node-rest` repository?

Reply:
good point, on one side we have this wrapped version and want to show examples for it how to use it, on the other side it is plainly using the wrapped rest package.

i prefer the former, so people don't need to know about the internals (`bfx-api-node-rest` is wrapped) and can use the official main api package out of the box. 

about moving: bfx-api-node-rest already has a few examples. the examples here should demonstrate how to use *this package here*, which is the main api package, which has support for all api endpoints.

Reply:
@vigan-abd any thoughts from your side how to manage that going forward in the future?

Reply:
Hi @robertkowalski @vitaliystoliarovcc , I would vote for using the class exported from this package, it is true that we're wrapping it, but what if in future we replace the class with an extended/different version from bfx-api-node-rest? Also since examples are related to this repo/lib I think it would be more convenient to use the classes/functions from this package